ambry-api/src/main/java/com.github.ambry/router/GetBlobOptions.java

```diff
@@ -19,16 +19,28 @@
  * {@link Router#getBlob(String, GetBlobOptions, Callback)} call.
  */
 public class GetBlobOptions {
+  public static final GetBlobOptions DEFAULT_OPTIONS = new GetBlobOptions(OperationType.All, null);
```

Comment:
rather than do this, perhaps you could expose a default constructor that initializes these default values?